packages/runtime/src/state/makeswift-api-client.ts

```diff
@@ -1,44 +1,98 @@
-import { applyMiddleware, createStore, Store as ReduxStore } from 'redux'
-import thunk, { ThunkAction, ThunkDispatch } from 'redux-thunk'
+import {
+  applyMiddleware,
+  createStore,
+  combineReducers,
+  type Dispatch as ReduxDispatch,
+  type Store as ReduxStore,
+  type Middleware,
+} from 'redux'
+import thunk, { type ThunkAction, type ThunkDispatch } from 'redux-thunk'
 
 import * as APIResources from './modules/api-resources'
-import { Action, apiResourceFulfilled } from './actions'
+import * as LocalizedResourcesMap from './modules/localized-resources-map'
+import { type Action, ActionTypes, apiResourceFulfilled, setLocalizedResourceId } from './actions'
 import {
-  APIResource,
   APIResourceType,
-  Swatch,
-  File,
-  Typography,
-  GlobalElement,
-  PagePathnameSlice,
-  Table,
-  LocalizedGlobalElement,
+  type APIResource,
+  type Swatch,
+  type File,
+  type Typography,
+  type GlobalElement,
+  type PagePathnameSlice,
+  type Table,
+  type LocalizedGlobalElement,
+  type APIResourceLocale,
 } from '../api'
 
-const reducer = APIResources.reducer
+const reducer = combineReducers({
+  apiResources: APIResources.reducer,
+  localizedResourcesMap: LocalizedResourcesMap.reducer,
+})
```

Comment:
Combine the `APIResources` and `LocalizedResourcesMap` states. We could move the `LocalizedResourcesMap` state into the `APIResources`, but this layered version felt cleaner to me.